.storybook/webpack.config.js

```diff
@@ -53,39 +41,6 @@ module.exports = {
           },
         ],
       },
-      {
```

Comment:
I put this in the description:
> Removed the *.css loader in the storybook config as this appears unused.

Happy to put back, but it does seem to be redundant.